src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -81,47 +81,27 @@ impl Renderer for HtmlHandlebars {
                         content = utils::render_markdown(&content);
                         print_content.push_str(&content);
 
-                        // Remove content from previous file and render content for this one
-                        data.remove("path");
-                        match ch.path.to_str() {
-                            Some(p) => {
-                                data.insert("path".to_owned(), p.to_json());
-                            },
-                            None => {
-                                return Err(Box::new(io::Error::new(io::ErrorKind::Other,
-                                                                   "Could not convert path to str")))
-                            },
-                        }
-
-                        // Remove content from previous file and render content for this one
-                        data.remove("content");
```

Comment:
I assume the calls to `remove` here were superfluous, because `insert` updates existing keys?
I can't recall why I added them in the first place, but if it works the same without them :+1: